test/AllInOne.Formatted.cs

```diff
@@ -749,6 +749,40 @@ public void ConstructedType()
             List<int> i = null;
             int c = i.Count;
         }
+
+        class Foo : IDisposable
+        {
+            public int N;
+
+            public void Dispose()
+            {
+            }
+        }
+
+        unsafe void EmptyEmbeddedStatment()
+        {
+            // useless empty statements get deleted
+
```

Comment:
Not critical, but a single new line would be enough.

Reply:
I didn't write this, this is the formatted file. 
This is a separate bug; empty statements get converted to empty lines, instead of printing nothing.

Reply:
Yes. I know it's the test result. I thought that was a bug introduced by your changes. And you're right, it's not.